javascript_client/src/subscriptions/ActionCableLink.ts

```diff
@@ -46,7 +46,7 @@ class ActionCableLink extends ApolloLink {
           )
         },
         received: function(payload) {
-          if (payload.result.data || payload.result.errors) {
+          if (payload?.result?.data || payload?.result?.errors) {
```

Comment:
In retrospect, maybe this should have been `if (payload.result)`, do you think that would work in your case?

Reply:
done!

Reply:
@rmosolgo could you run the CI and could we merge this if it's good?